modopt/tests/test_opt.py

```diff
@@ -627,6 +632,22 @@ def test_d_support_norm(self):
 
         npt.assert_raises(ValueError, proximity.KSupportNorm, 0.0, 0)
 
+    def test_group_lasso(self):
+        random_data_tile = np.tile(
+            self.random_data,
+            (self.group_lasso.weights.shape[0], 1, 1)
+        )
+        result_data = 2 * random_data_tile - 3
+        result_data = (result_data * (result_data > 0).astype('int')) / 2
```

Comment:
Sorry @chaithyagr, but this should also be in the `setUp`. To be consistent with the other tests, only the assertions should be in the test function.

Reply:
Ahh, yes. Moved.